File: backend/scripts/scripts/script-05.py
import json
from selenium import webdriver
from selenium.webdriver.common.keys import Keys
import time
from selenium.webdriver.chrome.options import Options

client = webdriver.Chrome()

client.get("https://www.google.com/")

client.find_element('xpath', '/html/body/div[1]/div[3]/form/div[1]/div[1]/div[1]/div/div[2]/input').send_keys('cotação dolar')

client.find_element('xpath', '/html/body/div[1]/div[3]/form/div[1]/div[1]/div[1]/div/div[2]/input').send_keys(Keys.ENTER)

time.sleep(10)

print(json.dumps('Código Executado 05!'))

File: backend/scripts/script-05.js
const { search } = require('../routes/automatizationRoutes');
const {Builder, Browser, By, Key, until} = require("selenium-webdriver")
const chrome = require('selenium-webdriver/chrome')
const chromedriver = require("chromedriver")

const service = () => chrome.setDefaultService(new chrome.ServiceBuilder(chromedriver.path).build());

teste = () => {

    return new Promise((resolve, reject) => {

        (async function example() {

            let driver = await new Builder().forBrowser(Browser.CHROME).build();
    
            try {
                await driver.get('https://www.google.com');
                const searchBar = await driver.findElement(By.xpath('/html/body/div[1]/div[3]/form/div[1]/div[1]/div[1]/div/div[2]/input'))
                await searchBar.sendKeys("cotação euro")
                await searchBar.sendKeys(Key.ENTER)
                
                const valueEuro = await driver.findElement(By.xpath('//*[@id="knowledge-currency__updatable-data-column"]/div[1]/div[2]/span[1]')).getText()
                resolve(valueEuro)
    
            } finally {
                setTimeout(() => {
                    driver.quit();
                }, 10000)
            }
        })
    
        ();
    })
}
module.exports = { teste };